packages/cli/src/Server.ts

```diff
@@ -1975,124 +1975,122 @@ class App {
 		// Authorize OAuth Data
 		this.app.get(
 			`/${this.restEndpoint}/oauth1-credential/auth`,
-			ResponseHelper.send(
-				async (req: OAuthRequest.OAuth1CredentialAuth, res: express.Response): Promise<string> => {
-					const { id: credentialId } = req.query;
-
-					if (!credentialId) {
-						throw new ResponseHelper.ResponseError(
-							'Required credential ID is missing',
-							undefined,
-							400,
-						);
-					}
+			ResponseHelper.send(async (req: OAuthRequest.OAuth1Credential.Auth): Promise<string> => {
```

Comment:
not sure why we need to touch the `oauth` section on this branch?

Reply:
If you mean the type, for boyscouting. Request types were added over multiple PRs and needed to be centralized and streamlined. Since it takes a bit for review and it is only type-related, I reasoned it'd be better include do this here.

If you mean the endpoint as a whole, I removed the unused arg `res` as I refactored the type, allowing us to reduce indentation in the whole endpoint. No logic changes.